tf2_ros/src/tf2_monitor.cpp

```diff
@@ -148,7 +142,7 @@ class TFMonitor
         buffer_._chainAsVector(frameb_, tf2::TimePointZero, framea_, tf2::TimePointZero, frameb_, chain_);
       }
       catch(tf2::TransformException& ex){
-        ROS_WARN("Transform Exception %s", ex.what());
```

Comment:
It looks like it could use the rclcpp logging macros `RCLCPP_WARN(node_->get_logger(), "...")`